lib/normalize_title_filter.js

```diff
@@ -64,9 +64,14 @@ module.exports = function(hyper, req, next, options, specInfo) {
                             location: contentLocation
                         }
                     });*/
-                    hyper.log('warn/normalize', {
-                        msg: 'Normalized requested title from ' + rp.title + ' to ' + result
-                    });
+
+
+                    // 5% chance of logging to bound volume.
+                    if (Math.random() < 0.05) {
+                        hyper.log('warn/normalize', {
+                            msg: 'Normalized requested title from ' + rp.title + ' to ' + result
```

Comment:
It would be really useful to have `from` and `to` fields as well, as it makes it much easier to inspect the logs (structured vs text)